source/_components/ambiclimate.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: page
+title: "Ambiclimate A/C controller"
+description: "Instructions on how to integrate Ambiclimate A/C controller into Home Assistant."
+date: 2019-03-21 15:00 +0200
```

Comment:
Please do not change the date.

Reply:
ok, why?

Reply:
it's only for when the page is created, not for when the component will be released 😉

Reply:
Ok, but I do not understand why it matters as long as it is a new file.